content/yaml-quick-start/building-a-flutter-app.md

```diff
@@ -57,10 +57,18 @@ Set up local properties
 ```yaml
 - echo "flutter.sdk=$HOME/programs/flutter" > "$FCI_BUILD_DIR/android/local.properties"
 ```
-### Building .apk
+### Building an app bundle
 
 ```yaml
-- flutter build apk --release
+- flutter build appbundle --release
+```
+
+In order to build an Android App Bundle (`.aab`), the command `./gradlew bundleRelease` is used, while for `.apk` creation `the command ./gradlew assembleRelease` is used. Note that Google Play requires new apps to be published as an Android App Bundle which replaces `.apk` as the standard publishing format. 
```

Comment:
just wondering - are these commands relevant to flutter apps? to generate an app bundle, it's enough to run the `flutter build appbundle --release` above

Reply:
indeed, these commands are not needed with flutter apps. removed them.